### PR TITLE
Plane: log TECS target alt

### PR DESCRIPTION
**Pull request by tridge**
log the target alt we pass into TECS to help debug a CRUISE height issue
for #21984

### CODE REVIEW

File: ArduPlane/Log.cpp [hunk 1]
```diff
@@ -312,16 +314,17 @@ const struct LogStructure Plane::log_structure[] = {
...
 // @Field: XT: the vehicle's current distance from the current travel segment
 // @Field: XTi: integration of the vehicle's crosstrack error
-// @Field: AspdE: difference between vehicle's airspeed and desired airspeed
+// @Field: AsE: difference between vehicle's airspeed and desired airspeed
 // @Field: TLat: target latitude
 // @Field: TLng: target longitude
-// @Field: TAlt: target altitude
-// @Field: TAspd: target airspeed
+// @Field: TAW: target altitude WP
+// @Field: TAT: target altitude TECS
```
Comment (IamPete1) [inline ArduPlane/Log.cpp:324]:
```suggestion
// @Field: TA: target altitude WP
// @Field: TA2: target altitude TECS
```

Comment (tridge) [reply ArduPlane/Log.cpp:324]: fixed

**Review by IamPete1 — APPROVED** [review]
LGTM

**Comment by shellixyz** [comment]
I just flew again this time it was going up every time I switched to cruise

image
[log_98_2022-10-18-19-11-18.bin.zip](https://github.com/ArduPilot/ardupilot/files/9813799/log_98_2022-10-18-19-11-18.bin.zip)

**Pull request merged**